src/promptflow/tests/executor/single_line_python_executor_proxy.py

```diff
@@ -0,0 +1,55 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from pathlib import Path
+from typing import Any, Mapping, Optional
+
+from promptflow._proxy._base_executor_proxy import AbstractExecutorProxy
+from promptflow.executor import FlowExecutor
+from promptflow.executor._line_execution_process_pool import LineExecutionProcessPool
+from promptflow.executor._result import LineResult
+from promptflow.storage._run_storage import AbstractRunStorage
+
+
+class SingleLinePythonExecutorProxy(AbstractExecutorProxy):
+    def __init__(self, flow_executor: FlowExecutor, line_execution_process_pool: LineExecutionProcessPool):
```

Comment:
Forgot to delete the original one in source code?

Reply:
Thanks, removed.